classes/Pods.php

```diff
@@ -1015,12 +1015,25 @@ public function field( $name, $single = null, $raw = false ) {
 						$field_names = explode( '.', $params->name );
 
 						if ( isset( $field_names[1] ) ) {
-							$size = $field_names[1];
+							$size    = $field_names[1];
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 4 spaces

<sub>hash: 06f43fba9aa0b9ffdfda99b095fe6188</sub>